### ISSUE TITLE
Warp Signs Restrictions

### ISSUE THREAD

**Issue by KingGoldensPanda** [BentoBoxWorld/addon-welcomewarpsigns#31]
When players place a wall sign, or a sign on top of a sign for example, for the warp, the warp will say it is unsafe. This is true in a sense, as there is no block directly under the sign, so it says it is unsafe, however it may actually be safe.
This causes issues as players do not like to be simple and use logic, and I have received 20+ complaints about it not letting them warp to it and saying its not safe, and they simply get confused and frustrated quickly.

The main solution I propose is:
 If they try to create a warp sign in one of the invalid locations, break the sign and tell them a message like, "&cYou can only place a Warp Sign on the ground."

A secondary solution could be just letting them warp there anyways, possibly also adding a message about placing it on the ground instead when they create it.

**Comment by tastybento** [comment]
Makes sense. I'll add a safety check on sign placement. It won't catch everything but it should help.
